bin/main.js

```diff
@@ -157,6 +157,9 @@ export async function main(nodeProcess) {
         }
         input = files.pop();
       }
+      if (!await fileExists(input)) {
+        throw Error(`Cannot load input file '${input}'`);
+      }
```

Comment:
Checking if a file exists right before reading it could lead to a race condition. The Node.js docs warn against this here:

> Using fs.exists() to check for the existence of a file before calling fs.open(), fs.readFile(), or fs.writeFile() is not recommended. Doing so introduces a race condition, since other processes may change the file's state between the two calls. Instead, user code should open/read/write the file directly and handle the error raised if the file does not exist.

https://nodejs.org/api/fs.html#fsexistspath-callback

Reply:
I've removed this exists call in https://github.com/markedjs/marked/pull/3165/commits/42bbb8604a0573814d464e11a000cf6d361dba58 to rely on the ENOENT error check instead.